ui/src/pages/OBProxy/index.tsx

```diff
@@ -53,17 +54,21 @@ export default function OBProxy() {
             handleAddCluster={handleAddCluster}
           />
         </Col>
-        <Col span={24}>
-          <EventsTable objectType="OBPROXY" />
-        </Col>
+        {access.systemread || access.systemwrite ? (
+          <Col span={24}>
+            <EventsTable objectType="OBPROXY" />
+          </Col>
+        ) : null}
       </Row>
-      <MonitorComp
-        filterLabel={[]}
-        queryScope="OBPROXY"
-        type="OVERVIEW"
-        groupLabels={['cluster']}
-        queryRange={DEFAULT_QUERY_RANGE}
-      />
+      {access.systemread || access.systemwrite ? (
```

Comment:
Why add `system` permission check here while remove the check in `pages/Cluster/index.tsx`?

Reply:
This is missing, the check for system should be removed.